### PR TITLE
Fixed a bunch of mypy errors

### PR DESCRIPTION
**Pull request by stanimirpehlivanov**
First ever pull request !

### CODE REVIEW

File: bin/install-dev [hunk 1]
```diff
@@ -9,7 +9,7 @@ set -e
 #   - must be run from project root
 # Compiles Protobuf files
 
-export PYTHONPATH="${PWD}:${PWD}/src/main/:${PWD}/src/test/:${PWD}/generated/"
+export PYTHONPATH="${PWD}:${PWD}/src:${PWD}/generated/"
```
Comment (stanimirpehlivanov) [inline bin/install-dev]: I think this was messing up the python config & imports

File: mypy.ini [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# MyPy config file
+# File reference here - http://mypy.readthedocs.io/en/latest/config_file.html#config-file
+
+[mypy]
+warn_redundant_casts = True
+warn_unused_ignores = True
+
+# Needed because of bug in MyPy
+disallow_subclassing_any = False
+ignore_missing_imports = True
```
Comment (stanimirpehlivanov) [inline mypy.ini:10]: This make's it stop complaining about 3rd party libs that dont have types

**Comment by apmaros** [comment]
also, can you please remove `.DS_Store `?

**Comment by apmaros** [comment]
Added pre-commit hooks to do some checks and black with mypy can be added there

**Comment by stanimirpehlivanov** [comment]
> also, can you please remove `.DS_Store `?

Hmm, this makes me think it may be useful to check-in .gitignore in the repo, so that anyone that pulls it will not be checking in unwanted files

**Comment by apmaros** [comment]
@S-Pehlivanov you could also add it to global `.gitignore`

**Pull request merged**